_posts/2024-03-22-release.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Release 2024.12
+author: Priya
+date: 2024-03-22
+category: Jekyll
+layout: post
+---
+### Defect Fixes:
+* Resolved multiple issues in join rule including prefix addition in renamed columns and consistency between foreign and parent views.
+* Fixed dataflow map overlay size issue for better display.
```

Comment:
Make this an enhancement in an Enhancement section.
On clicking expand, Data flow map now expands to the full browser size.

Reply:
fixed